### PR TITLE
Ensure dev-setup.sh works on minimal RHEL8 as well

### PR DESCRIPTION
**Pull request by SEJeff**
* use minikube 1.24 due to [this nasty bug](https://github.com/kubernetes/minikube/issues/13542)
* support running on Debian 10, Debian 11, and RHEL8 or a derivative

Note that 69255572c358fb52d4b83ec537cf69e38423cda8 was written entirely by @kierdavis so the credit or flames are all hers :)

---
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/875)

### CODE REVIEW

File: scripts/dev-setup.sh [hunk 1]
```diff
@@ -25,18 +29,32 @@ if ! sudo -n true; then
...
+case "$DISTRO" in
+    Debian-*)             sudo apt-get update && sudo apt-get upgrade -y ;;
+    RedHatEnterprise-8.*) sudo dnf upgrade -y ;;
+    *) echo "Internal error: $DISTRO not matched in case block." && exit 1 ;;
+esac
 
 # Install dependencies
-sudo apt-get -y install bash-completion git git-review vim
+case "$DISTRO" in
+    Debian-*)             sudo apt-get -y install bash-completion git git-review vim ;;
```
Comment (jynnantonix) [inline scripts/dev-setup.sh:54]: We may want to consider adding kubectl to this list as well.  It's mentioned in DEVELOP.md but doesn't get installed by anything in this script.

**Review by jynnantonix — APPROVED** [review]

File: scripts/dev-setup.sh (hunk 1 above)
Comment (leoluk) [reply scripts/dev-setup.sh:54]: The install script adds an alias: https://github.com/certusone/wormhole/blob/885754e723cc7b9c136ceb37471ece8b50e9cff6/scripts/dev-setup.sh#L93

Comment (jynnantonix) [reply scripts/dev-setup.sh:54]: Ah, I see.  I use fish so bash aliases have no effect for me.  I'll look more carefully next time.

**Review by leoluk — DISMISSED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/875)* status: all files reviewed, 2 unresolved discussions (waiting on @jynnantonix and @SEJeff)

**Review by jynnantonix — APPROVED** [review]

**Review by jynnantonix — DISMISSED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/875)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @SEJeff)

**Review by jynnantonix — APPROVED** [review]
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/875)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @leoluk)

**Review by leoluk — APPROVED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/875)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @SEJeff)

**Pull request merged**